### DISCUSSION TITLE
Managing parasite and host collections in Arctos

### DISCUSSION THREAD

**Discussion by campmlc** [ArctosDB/arctos#8288] [General]
This discussion is a forum for addressing parasite and host management in Arctos. 
@campmlc @gracz-UNL @bryansmclean @keg34 @jrpletch @acdoll @jebrad @andrew-hope @LitsaWoo @jweckstein @ebraker @kderieg322079 @DerekSikes @adhornsby @msbparasites @jldunnum @AdrienneRaniszewski

**Comment by campmlc** [comment]
Some recent topics raised in discussions:


I'm aware of 6(!) ways to search for parasites on Arctos which all can give different results:
Searching the Mammal catalog for parasite parts (this is what I've been typically starting with) - searching for MSB:Mamm for Ochotona fleas gives 93 entries
Searching the Parasite catalog for records with a "host of" relationship (which should work if both the parasites and mammals are in Arctos) - this returns only 54 entries
Searching for the Parasite catalog for verbatim host ID (which unfortunately isn't integrated with the taxonomy) - only 21 records
Searching the Mammal catalog for those that have a "parasite of" relationship - 18 results
Searching the Host catalog for those that have a "parasite of" relationship - 4 results
Searching the Host catalog for those that have parasite parts - 0 results

**Reply by campmlc** [reply]
MSB mammal parasites are first cataloged as records in the MSB:Mamm collection, and then subsequently transferred to our parasite division catalog, which has only existed for the past decade. Most legacy mammal parasite records will still be found by searching the mammal records.  Because there are over two decades worth of parasite data in the mammal collection, these data pre-date the existence of Arctos parasite collections and data fields. Therefore these older legacy data may be found in other fields such as mammal record and part remarks. 
Newer field collections and record types (e.g. ectoparasites) that have had dedicated funding for data migration will have mammal records and parasite records both cataloged and linked by "parasite of" and "host of" relationships between Arctos record guids. But there is a temporal lag in both cataloging and in creation of the reciprocal linkages.  The mammal record usually is cataloged first, without a related parasite record but with parasite "parts" and "examined/detected" attributes. The parasite record is cataloged later, usually but not always after parasite sorting and identification via loans or other projects. The parasite record adds in "verbatim host ID" when there is no immediately available linkage to a host record. Otherwise, if there is a cataloged mammal host, the parasite record will be directly linked to the mammal host through Arctos relationships. The reverse reciprocal linkage between host and parasite  comes later through automated Arctos tools, or curatorially through identifier bulkloads. 
When the model for parasite collections was first developed for Arctos in 2011, we had to address the issue of how to make related parasites and hosts searchable and discoverable when physical vouchers either were not in collection or even known to have been preserved. This is the reason for the creation of a Host catalog, e.g. MSB:Host. It primarily acts to hold observational data for hosts which may not exist as physical vouchers in any museum, or whose disposition/collection is unknown or pending discovery. Unlike the parasite record attribute "verbatim host ID", which is free-text and uncontrolled vocabulary and thereby not useful for discovery, the Host collection records embed the host identification in higher taxonomy, so it becomes possible to perform searches such as "find all fleas of family X that are parasites of host species Y".  If an actual host voucher exists, however, the link to the host will be through the Mammal collection portal (or whatever other taxonomic division of interest), if the reciprocal relationship link has been created. If these links exist between any related records across any collections in Arctos, the mutual relationships will be discoverable.
So in answer to your question above - 
1) searching the mammal record for parasites as parts should give the most complete list for current collections, because of the lag time in creating parasite records. 
2) Mammal records should also be searched in attribute (examined/detected) and record remarks fields, as legacy data are still stored here pending funding to migrate this information.
3) Searching the parasite catalog for host relationships will give fewer results if the parasite record does not exist yet or if either the parasite of relationship to the host or the host of relationship to the parasite have not yet been added to the record.
4) Searching the mammal catalog for related records with a  "parasite of" relationship - again, fewer results if reciprocal links have not all been created.
5) Searching the Host catalog: only works for legacy "host" data that did not have or has not been linked to a corresponding Mammal division record. The majority of the mammals in this catalog are Robert Rausch hosts that were entered as observations from his field ledgers. As such, these records do not have "parts", as they are based on media transcriptions.

As you can see, it takes a lot of resources to make all these connections, and we could use a lot more staffing and dedicated funding for legacy data capture and also data discoverability!

**Reply by campmlc** [reply]
Note that the reverse process applies for parasite collections that catalog material and then subsequently create linkages to host collections - or for any types of related records in Arctos. It is unusual for hosts and parasites to be created simultaneously - there is usually always a lag, as different collections have different workflows, permissions, and resources.

**Comment by campmlc** [comment]
Hi Mariel, a couple other things I thought of:
Some of the fleas had mites (see image attached). In some cases, they had fallen off the fleas and were floating in the ethanol. When I found them, I put them in their own vial. How would you handle that in Arctos? Would you have in the flea parasite record, one part that is just ectoparasite with mites in the remarks and then make that a daughter of the whole organism part?

On a similar theme, I've been working with some tapeworm whole genome data and managed to extract the host mitogenome from one of the specimens using Novoplasty. If I were to upload this to genbank, how would you recommend linking it to Arctos? Would you attach it to the host record, the parasite record, or both? 
from @jrpletch

**Comment by campmlc** [comment]
Zoom call scheduled for Friday Nov 22. Please join.

**Comment by Jegelewicz** [comment]
@campmlc can I come along? I have some host/parasite stuff in the UWBM data and I want to handle it as efficiently as possible.

**Comment by ebraker** [comment]
@jrpletch We have jars of ethanol-preserved shrews and voles that were examined for parasites. The shrews/voles in each jar were often from the same pitfall trap or at least collected along the same transect. Though it was unclear which parasite fell from which individual, we ended up creating entities for each jar, since we could at least narrow the host to a handful of individuals from essentially the same site and felt that the relationship was still meaningful to illustrate. See https://arctos.database.museum/search.cfm?guid_prefix=Arctos%3AEntity&cat_num=34-39 for examples.
